### ISSUE TITLE
Links to example RISE talks

### ISSUE THREAD

**Issue by choldgraf** [damianavila/RISE#263]
It might be useful if there were a mini "gallery" of videos of talks that had been given with RISE, or slide decks that have used RISE and are publicly available.

I bet we could build in RISE compatibility with Binder somehow. That way a repo could become a public, interactive presentation. A gallery could be a good way to help people discover RISE and see what it's capable of. Just a thought for future reference!

**Comment by damianavila** [comment]
> It might be useful if there were a mini "gallery" of videos of talks that had been given with RISE, or slide decks that have used RISE and are publicly available.

Yep, I am pretty sure there are a lot of talks out there using RISE. I think it would be nice to have that "gallery".

> I bet we could build in RISE compatibility with Binder somehow. That way a repo could become a public, interactive presentation. A gallery could be a good way to help people discover RISE and see what it's capable of. Just a thought for future reference!

This is a great idea!!

**Comment by jochym** [comment]
Actually some of the materials on our workshop (https://github.com/jochym/abinitio-workshop - unfortunately in polish so not useful for the gallery) are exactly that way. And they run on binder nicely.

**Comment by choldgraf** [comment]
Very cool!
